assets/js/googlesitekit/notifications/register-defaults.js

```diff
@@ -453,18 +454,10 @@ export const DEFAULT_NOTIFICATIONS = {
 		groupID: NOTIFICATION_GROUPS.SETUP_CTAS,
 		viewContexts: [ VIEW_CONTEXT_MAIN_DASHBOARD ],
 		checkRequirements: async ( { select, resolveSelect } ) => {
-			if ( ! isFeatureEnabled( 'firstPartyMode' ) ) {
-				return false;
-			}
+			const fpmModuleConnected =
```

Comment:
A minor point, but please can you update this to be consistent with the surrounding boolean identifiers?

```suggestion
			const isFPMModuleConnected =
```